### ISSUE TITLE
Time limited avalanchemqperf througput consumer tag warning

### ISSUE THREAD

**Issue by jage** [cloudamqp/lavinmq#297]
```
$ avalanchemqperf throughput -z 1
2021-10-27T12:18:33.863910Z   WARN - amqp.client.channel: Consumer tag 'c' already cancelled -- channel_id: 1, fiber: "read_loop"
2021-10-27T12:18:33.863910Z   WARN - amqp.client.channel: Consumer tag 'c' already cancelled -- channel_id: 1, fiber: "read_loop"
... # hundreds of rows
Publish rate: 1 msgs/s Consume rate: 10504 msgs/s

Summary:
Average publish rate: 212161.5 msgs/s
Average consume rate: 211942.1 msgs/s
```

**Comment by jage** [comment]
Issue in amqp-client.cr, change from warning to debug.

**Comment by carlhoerberg** [comment]
Reverted to debug logging with https://github.com/cloudamqp/amqp-client.cr/commit/e98106808f1b2930dc07bfe909b930216864deee

**Comment by jage** [comment]
Fix released with 1.0.0-beta.1
